05-Organizing-and-Executing-Simulations/01-Organizing-Data.ipynb

```diff
@@ -0,0 +1,334 @@
+   "source": [
+    "import os\n",
+    "\n",
+    "fn = os.path.join(os.getcwd(), 'signac.rc')\n",
+    "![ -e \"$fn\" ] && rm \"$fn\"\n",
+    "fn = os.path.join(os.getcwd(), 'dataspace')\n",
+    "![ -e \"$fn\" ] && rm -rf \"$fn\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Research question\n",
+    "\n",
+    "The [Introducing HOOMD-blue](../00-Introducing-HOOMD-blue/00-index.ipynb) tutorial shows how to execute a single simulation of hard octahedra and how they self-assemble into a crystal structure at a high volume fraction.\n",
+    "At what volume fraction is the phase transition from fluid to crystal?\n",
+    "One way to find out is to execute simulations at many volume fractions and examine the resulting equilibrium structures.\n",
+    "When performing such a study, you may want to explore simulations at different system sizes, repeat the simulation with different random number seeds, or examine the effects of changing other parameters.\n",
+    "\n",
+    "The unique set of parameters for each simulation is a **statepoint** which you can represent in a Python dictionary:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'N_particles': 128, 'volume_fraction': 0.57, 'seed': 20}"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "statepoint = dict(N_particles=128, volume_fraction=0.57, seed=20)\n",
+    "statepoint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In your own research, you will execute different types of simulation with different parameters - but you will have the same need to organize and execute many simulations.\n",
+    "Follow the example provided in this tutorial and apply the same concepts to your work."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data space"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Each simulation you execute will generate several files of output.\n",
+    "Store these in a **directory** uniquely assigned to each **statepoint**.\n",
+    "The collection of **directories** is a **data space**.\n",
+    "Use [**signac**](https://docs.signac.io/en/latest/) to automate the process of creating the **directories**."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import signac"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**signac** stores a **data space** in a project.\n",
+    "Create one:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "project = signac.init_project(name=\"dataspace\", workspace=\"dataspace\")"
```

Comment:
```suggestion
    "project = signac.init_project(name=\"octahedra-assembly-project\")"
```

Reply:
I am naming the directory `dataspace` here to avoid a fluid representation where "workspace" is used as an alternate name to "dataspace".

Reply:
Yes, but likely soon (`signac` 2.0 this won't be an option, and we will change it anyways.